products/neo-savant/Makefile

```diff
@@ -0,0 +1,81 @@
+.PHONY: all
+all: help
+
+.ONESHELL:
+SHELL := /bin/bash
+.SHELLFLAGS = -ec
+
+ENVIRONMENT ?= dev
+VALID_ENVIRONMENTS := dev stg prd
+IMAGE_TAG ?= localhost:5001/neo-savant:latest
+
+# Check if the ENVIRONMENT variable is in the list of valid environments
+ifeq ($(filter $(ENVIRONMENT),$(VALID_ENVIRONMENTS)),)
+$(error Invalid value for ENVIRONMENT. Valid values are dev, stg, or prd.)
+endif
+
+# Ensures that a variable is defined and non-empty
+define assert-set
+	@$(if $($(1)),,$(error $(1) not defined in $(@)))
+endef
+
+DEVOPS_REPO_URL := https://$(GITHUB_PAT)@github.com/Zilliqa/devops.git
+TEMP_FOLDER := $(if $(DEVOPS_FOLDER),$(DEVOPS_FOLDER),.z)
+
+devops_repo_cleanup:
+	@if [ -z "$(DEVOPS_FOLDER)" ]; then \
+		rm -rf $(TEMP_FOLDER); \
+	fi
+
+## Download or use the given devops repo to use z locally
+z/setup: devops_repo_cleanup
```

Comment:
Why do you need to do this? Again, if you're using 'z app', you should be able to just build your container and go ..